liquibase-standard/src/main/java/liquibase/Liquibase.java

```diff
@@ -67,6 +69,10 @@ public class Liquibase implements AutoCloseable {
     private final DefaultChangeExecListener defaultChangeExecListener = new DefaultChangeExecListener();
     private final Map<String, Boolean> upToDateFastCheck = new HashMap<>();
 
+    @Getter
+    @Setter
+    private Boolean fast_check_disabled = Boolean.FALSE;
```

Comment:
Consider using positive - `fastCheckEnabled` to make it easier to understand if it is enabled or disabled.

Reply:
Yeah, I can rename it if we want to move forward with this PR. I thought calling it this way (`fast_check_disabled`) because by default this configuration would stay active, but yes I'm fine renaming it.